### PR TITLE
[Stream] Tidying up Stream live video content

### PR DESCRIPTION
**Pull request by dcpena**
Tidying up Stream live video content. Mostly reorganizing content, adding additional info, or adding in headers.

Addresses PCX-4493.

### CODE REVIEW

File: content/stream/stream-live/_index.md [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
 pcx_content_type: navigation
-weight: 6
+weight: 4
 ---
 
-# Stream Live
+# Stream live video
 
-{{<directory-listing>}}
+Use Cloudflare Stream to stream live video and broadcast your own live events or add live video to your own app.  When you use Stream, you get low latency without having to worry about encoding or ingestion fees, finding a CDN, or modifying your configuration; Stream takes care of this for you.
```
Comment (kodster28) [inline content/stream/stream-live/_index.md]:
```suggestion
Use Cloudflare Stream to stream live video and broadcast your own live events or add live video to your own app. When you use Stream, you get low latency without having to worry about encoding or ingestion fees, finding a CDN, or modifying your configuration; Stream takes care of this for you.
```

File: content/stream/stream-live/replay-recordings.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
+pcx_content_type: how-to
+title: Replay recordings
+weight: 4
+---
+
+# Replay recordings
+
+Live streams are automatically recorded by default. To get a list of recorded streams for a given input ID, make the same `GET` request as you would to get the live video and filter for videos where the state property is set to `ready`:
```
Comment (kodster28) [inline content/stream/stream-live/replay-recordings.md]: Maybe include a link out to api docs if there's a section for that?

File: content/stream/stream-live/start-stream-live.md [hunk 3]
```diff
@@ -8,21 +8,21 @@ weight: 1
 
 You can start a live stream using the Stream dashboard or the API. After you subscribe to Stream, you can create Live Inputs and begin sending your live video to Cloudflare Stream using RTMPS or SRT. SRT supports newer video codecs and makes using accessibility features, such as captions and multiple audio tracks, easier.
 
-## Using the dashboard
```
Comment (kodster28) [inline content/stream/stream-live/start-stream-live.md:11]: Maybe worth migrating to using the new tabs component for api/dash split?

https://github.com/cloudflare/cloudflare-docs/blob/production/content/load-balancing/how-to/create-pool.md?plain=1#L19

File: content/stream/stream-live/webhooks.md [hunk 4]
```diff
@@ -7,11 +7,25 @@ weight: 6
...
+## Subscribe to Stream Live Webhooks
 
-The webhook payload will look similar to this:
+1.  Log in to your Cloudflare account and click **Notifications**.
+2.  From the **Notifications** page, click the **Destinations** tab.
+3.  On the **Destinations** page under **Webhooks**, click **Create**.
+4.  Enter the information for your webhook and click **Save and Test**.
+5.  To create the notification, from the **Notifications** page, click the **All Notifications** tab.
+6.  Next to **Notifications**, click **Add**.
+7.  Under the list of products, locate **Stream** and and click **Select**.
```
Comment (kodster28) [inline content/stream/stream-live/webhooks.md]:
```suggestion
7.  Under the list of products, locate **Stream** and click **Select**.
```

File: content/stream/stream-live/webhooks.md [hunk 5]
```diff
@@ -7,11 +7,25 @@ weight: 6
...
+8.  Enter a name and optional description.
+9.  Under **Webhooks**, click **Add webhook** and click your newly created webhook.
+10.  Click **Next**.
```
Comment (kodster28) [inline content/stream/stream-live/webhooks.md]:
```suggestion
10. Click **Next**.
```

File: content/stream/stream-live/webhooks.md [hunk 6]
```diff
@@ -7,11 +7,25 @@ weight: 6
...
+11.  By default, you will receive webhook notifications for all Live Inputs. If you only wish to receive webhooks for certain inputs, enter a comma delimited list of Input IDs in the text field.
```
Comment (kodster28) [inline content/stream/stream-live/webhooks.md]:
```suggestion
11.  By default, you will receive webhook notifications for all Live Inputs. If you only wish to receive webhooks for certain inputs, enter a comma-delimited list of Input IDs in the text field.
```

File: content/stream/viewing-videos/watch-live-stream.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+pcx_content_type: tutorial
+title: Watch a live stream
+weight: 3
+---
+
+# Watch a live stream
+
+When an input begins receiving the live stream, a new video with HLS and DASH URLs is automatically created as long as the mode property for the input is set to `automatic`.
+
+## Use the API
```
Comment (kodster28) [inline content/stream/viewing-videos/watch-live-stream.md]: Same as above... not sure if you want to format in tabs, but think this'd work. Probably would want to have the API as the first (and therefore default) tab

**Review by kodster28 — APPROVED** [review]

File: content/stream/stream-live/_index.md [hunk 8]
```diff
@@ -1,9 +1,25 @@
...
 pcx_content_type: navigation
-weight: 6
+weight: 4
 ---
 
-# Stream Live
+# Stream live video
 
-{{<directory-listing>}}
+Use Cloudflare Stream to stream live video and broadcast your own live events or add live video to your own app. When you use Stream, you get low latency without having to worry about encoding or ingestion fees, finding a CDN, or modifying your configuration; Stream takes care of this for you.
```
Comment (irvinebroque) [inline content/stream/stream-live/_index.md]:
```suggestion
Cloudflare Stream lets you or your users stream live video, and play live video in your website or app, without managing and configuring any of your own infrastructure.
```

File: content/stream/stream-live/_index.md [hunk 9]
```diff
@@ -1,9 +1,25 @@
...
+
+## How Stream works
+
+After you set up a live input, Stream works by using your stream URL and Key along with your broadcasting system to live stream your video. Refer to the diagram below to learn how each item interacts with each other to produce a live stream.
```
Comment (irvinebroque) [inline content/stream/stream-live/_index.md]:
```suggestion
Stream handles video streaming end-to-end, from ingestion through delivery.

1. For each live stream, you create a unique live input, either using the Stream Dashboard or API.
2. Each live input has a unique Stream Key, that you provide to you the creator who is streaming live video.
3. Creators use this Stream Key to broadcast live video to Cloudflare Stream, over either RTMPS or SRT.
4. Cloudflare Stream encodes this live video at multiple resolutions and delivers it to viewers, using Cloudflare's Global Network. You can play video on your website using the [Stream Player](/stream/viewing-videos/using-the-stream-player/) or using [any video player that supports HLS or DASH](/stream/viewing-videos/using-own-player/).

```

File: content/stream/stream-live/_index.md [hunk 10]
```diff
@@ -1,9 +1,25 @@
...
+
+## Latency
+
+You can use different streaming protocol options with Stream. Stream supports RTMPS and SRT.
+
+**RTMPS and SRT**
+- Provides 10+ second latency.
+- RTMP is ideal when broadcasting to users with low bandwith.
+- SRT is ideal for broadcasting high quality video to users with unreliable networks.
+- Both options allow you to record video for later playback.
```
Comment (irvinebroque) [inline content/stream/stream-live/_index.md]: Let's remove this section for now (can share context out of comments)

File: content/stream/stream-live/replay-recordings.md (hunk 2 above)
Comment (irvinebroque) [inline content/stream/stream-live/replay-recordings.md]:
```suggestion
Live streams are automatically recorded, and available instantly once a live stream ends. To get a list of recordings for a given input ID, make a `GET` request to `/live_inputs/<UID>/videos` and filter for videos where `state` is set to `ready`:
```

File: content/stream/stream-live/replay-recordings.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
+---
+pcx_content_type: how-to
+title: Replay recordings
+weight: 4
+---
+
+# Replay recordings
```
Comment (irvinebroque) [inline content/stream/stream-live/replay-recordings.md]:
```suggestion
# Record and replay live streams
```

File: content/stream/stream-live/simulcasting.md [hunk 12]
```diff
@@ -52,18 +56,12 @@ By default, all live outputs are enabled.
...
-
-To delete an output, make a `DELETE` request to the `/outputs` endpoint with the output id:
-
-```bash
-curl -X DELETE \ -H "Authorization: Bearer <API_TOKEN>" \https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/stream/live_inputs/<INPUT_UID>/outputs/<OUTPUT_UID>
+| Command      | Method | Endpoint|
+|--------------|--------|---------|
+| [List outputs](https://api.cloudflare.com/#stream-live-inputs-list-live-inputs)| `GET` | `accounts/:account_identifier/stream/live_inputs`|
+| [Delete outputs](https://api.cloudflare.com/#stream-live-inputs-delete-a-live-input)| `DELETE` |  `accounts/:account_identifier/stream/live_inputs/:live_input_identifier`|
 ```
```
Comment (irvinebroque) [inline content/stream/stream-live/simulcasting.md]:
```suggestion
```

File: content/stream/stream-live/start-stream-live.md (hunk 3 above)
Comment (irvinebroque) [reply content/stream/stream-live/start-stream-live.md:11]: I've experimented with this a bit and have a few design thoughts around making it more discoverable and prominent, before we start using it for bigger navigation things like this. Love that we have it for code samples in different languages though!

File: content/stream/stream-live/webhooks.md [hunk 13]
```diff
@@ -23,21 +37,5 @@ The webhook payload will look similar to this:
   },
   "ts": 1642074233
 }
```
Comment (irvinebroque) [inline content/stream/stream-live/webhooks.md:39]:
```suggestion
}
```
```

File: content/stream/changelog.md [hunk 14]
```diff
@@ -119,7 +119,7 @@ You can now use Cloudflare Stream for 24/7 live streaming.
 
 ### Persistent Live Stream IDs
 
-You can now start and stop live broadcasts without having to provide a new video UID to the Stream Player (or your own player) each time the stream starts and stops. [Read the docs](/stream/stream-live/watch-live-stream/#view-by-live-input-id).
+You can now start and stop live broadcasts without having to provide a new video UID to the Stream Player (or your own player) each time the stream starts and stops. [Read the docs](/stream/viewing-videos/watch-live-stream/#view-by-live-input-id).
```
Comment (irvinebroque) [inline content/stream/changelog.md]: Looking at this in the preview, I think we need to keep `Watch live streams` in its current spot, within the `Stream live video` section.

File: content/stream/viewing-videos/watch-live-stream.md [hunk 15]
```diff
@@ -0,0 +1,144 @@
...
+curl -X GET \
+-H "Authorization: Bearer <API_TOKEN>" \
+https://customer-m033z5x00ks6nunl.cloudflarestream.com/6b6972f427f51793099c6b427783398e/lifecycle
+```
+
+In the example below, the response indicates the `ID` is for an input with an active `videoUID`. The `live` status value indicates the input is actively streaming.
+
+```json
+---
+header: Response for active live stream
```
Comment (irvinebroque) [inline content/stream/viewing-videos/watch-live-stream.md]:
```suggestion
header: Response for an active live stream
```

File: content/stream/viewing-videos/watch-live-stream.md [hunk 16]
```diff
@@ -0,0 +1,144 @@
...
+    "videoUID": "55b9b5ce48c3968c6b514c458959d6a",
+    "live": true
+}
+```
+
+The example below confirms the ID is for an input but is not actively streaming.
+
+```json
+---
+header: Response for inactive live stream
```
Comment (irvinebroque) [inline content/stream/viewing-videos/watch-live-stream.md]:
```suggestion
header: Response for an inactive live stream
```

File: content/stream/viewing-videos/watch-live-stream.md [hunk 17]
```diff
@@ -0,0 +1,144 @@
...
+header: Response for active live stream
+---
+{    
+    "isInput": true,
+    "videoUID": "55b9b5ce48c3968c6b514c458959d6a",
+    "live": true
+}
+```
+
+The example below confirms the ID is for an input but is not actively streaming.
```
Comment (irvinebroque) [inline content/stream/viewing-videos/watch-live-stream.md]:
```suggestion
```

Think the headers capture this already

File: content/stream/viewing-videos/watch-live-stream.md [hunk 18]
```diff
@@ -0,0 +1,144 @@
...
+header: Response for inactive live stream
+---
+{
+    "isInput": true,
+    "videoUID": null,
+    "live": false
+}
+```
+
+When viewing a livestream via the live input ID, the `requireSignedURLs` and `allowedOrigins` options in the live input recording settings are used. These settings are independent of the video-level settings.
```
Comment (irvinebroque) [inline content/stream/viewing-videos/watch-live-stream.md]:
```suggestion
When viewing a live stream via the live input ID, the `requireSignedURLs` and `allowedOrigins` options in the live input recording settings are used. These settings are independent of the video-level settings.
```

File: content/stream/changelog.md (hunk 14 above)
Comment (irvinebroque) [reply content/stream/changelog.md]: I think there are good ways to link to this content _from_ the `Play video` section, but when I see it here in practice, it makes it seem like Watch a live stream is an entirely distinct path from using your own player, using the Stream Player, etc.

Screen Shot 2022-09-20 at 9 49 06 PM

...and when it's not in the `Stream live video` section, I wonder where I can find it...

Screen Shot 2022-09-20 at 9 49 40 PM

File: content/stream/stream-live/replay-recordings.md (hunk 2 above)
Comment (dcpena) [reply content/stream/stream-live/replay-recordings.md]: Done

File: content/stream/stream-live/_index.md (hunk 10 above)
Comment (dcpena) [reply content/stream/stream-live/_index.md]: Done

File: content/stream/changelog.md (hunk 14 above)
Comment (dcpena) [reply content/stream/changelog.md]: Moved back!

**Pull request merged**